src/qt/choice.cpp

```diff
@@ -10,38 +10,76 @@
 
 #include "wx/choice.h"
 #include "wx/qt/private/winevent.h"
+
 #include <QtWidgets/QComboBox>
+#include <QtCore/QSortFilterProxyModel>
 
-class wxQtChoice : public wxQtEventSignalHandler< QComboBox, wxChoice >
+namespace
 {
-public:
-    wxQtChoice( wxWindow *parent, wxChoice *handler );
+    class LexicalSortProxyModel : public QSortFilterProxyModel
+    {
+        public:
+            bool lessThan( const QModelIndex &left, const QModelIndex &right ) const wxOVERRIDE
```

Comment:
As part of the [Coding Guidelines](https://www.wxwidgets.org/develop/coding-guidelines/), pretty much the use of all spaces is well defined. People don't read them or at least they are not followed, and are seldomly enforced, but is it too big of a burden to at least try to follow them in new code?
i.e.
```suggestion
            bool lessThan(const QModelIndex& left, const QModelIndex& right) const wxOVERRIDE
```
as very nicely exemplified [here](https://www.wxwidgets.org/develop/coding-guidelines/#no_overloaded_virtuals).